README.md

```diff
@@ -103,6 +90,16 @@ Optionally, you can add this step also for more scripts, for example before star
 
 In general, this doesn't add to much overhead, since usually relative-deps is able to determine rather quickly (~0.5 sec) that there are no changes.
 
+### Configuration alternatives
+
+You can configure `relative-deps` in few ways, depends on your prefference:
```

Comment:
```suggestion
You can configure `relative-deps` in several ways, depending on your preference:
```

Reply:
I'd also add a link to `cosmiconfig` here cause it is so awesome and deserves credit :)